### ISSUE TITLE
MapPropertiesRefactoring encountered an error and has been disabled

### ISSUE THREAD

**Issue by spixy** [jeuxjeux20/EntityFrameworkRocket#1]
Hi, I got this error when editing `ConfigureServices` in Startup.

```
System.NullReferenceException : Object reference not set to an instance of an object.
   at EntityFrameworkRocket.Refactorings.MapPropertiesRefactoring.GetProperties(ObjectCreationExpressionSyntax objectCreation,ITypeSymbol newExpressionType,ITypeSymbol lambdaParameterType)
   at async EntityFrameworkRocket.Refactorings.MapPropertiesRefactoring.ComputeRefactoringsAsync(<Unknown Parameters>)
   at async Microsoft.CodeAnalysis.CodeRefactorings.CodeRefactoringService.GetRefactoringFromProviderAsync(<Unknown Parameters>)
```

**Comment by DynamicField** [comment]
Weird, this doesn't happen on my side but only with an outdated version, maybe that 2bca89f fixes this in version 1.3.1 which should be available now.

**Comment by DynamicField** [comment]
Oh nevermind i was able to reproduce the issue.
